src/openlifu/io/ustx.py

```diff
@@ -403,7 +404,7 @@ def get_delay_control_registers(self, profile: Optional[int]=None) -> Dict[int,i
         delay_profile = self.get_delay_profile(profile)
         apod_register = 0
         for i, apod in enumerate(delay_profile.apodizations):
-            apod_register = set_register_value(apod_register, 1-apod, lsb=i, width=1)
+            apod_register = set_register_value(apod_register, 1-apod, lsb=APODIZATION_CHANNEL_ORDER.index(i+1), width=1)
```

Comment:
@georgevigelette @peterhollender Please check this change introduced into `main` from `p/sonication_control_refactor`. Do we want this?